drivers/soundwire/bus.c

```diff
@@ -1047,14 +1047,16 @@ static int sdw_initialize_slave(struct sdw_slave *slave)
 	int ret;
 	u8 val;
 
+	dev_err(slave->bus->dev,
+		"programming SDW_SCP_INTMASK1\n");
```

Comment:
I'm a bit confused: first this doesn't seem to belong to this PR. And as a side note for this commit: this error message would confuse me, if I saw it in my kernel log. And it's supposed to appear every time SDW is used?

Reply:
This is to expose a problem with the other log, that shows whether we program the parity or not. It's not final of course, I just want to root-cause why we see parity errors for no good reason.